templates/blog/article_list.html

```diff
@@ -1,15 +1,23 @@
 {% from 'blog/macros.html' import article_list with context %}
 {% extends 'layout.html' %}
 
-{% block title %}Nomad articles - {{ articles[0].title|truncate(40) }}{% endblock %}
+  {% block title %}Nomad articles
```

Comment:
The things is that I don't succeed to put an if around a block for not displaying it 
For instance 
```
{% block extra_og %}
    {% if articles[0] is defined %}
        <meta property=og:image content="{{ articles[0].image }}">
        <meta property=og:title content="Latest articles ({{ articles[0].title }})">
        <meta property=og:description content="{{ articles[0]|striptags|truncate(200) }}">
    {% endif %}
{% endblock %}
```
If I put the `if` around the `block extra_og` I have a bug. And its the same with other blocks.
I haven't find a way to surround all the metas with only one `{% if articles[0] %} {% endif %}`

And do you prefer `{% if articles[0] %} {% endif %}` or `{% if articles[0] is defined %} {% endif %}`?